src/components/Avatar/connector.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux'
+import { showPopoverHandler } from '../../actions'
+
+export function mapDispatchToProps (dispatch) {
+  return {
+    onMouseOver: showPopoverHandler(dispatch)
```

Comment:
Fully agree-- good catch. I was in a tug of war with that one and in a few other cases, then settled on the same conclusion, but neglected to changing this one back.